SecurityPkg/Library/DxeTcg2PhysicalPresenceLib/DxeTcg2PhysicalPresenceLib.c

```diff
@@ -193,6 +195,19 @@ Tcg2ExecutePhysicalPresence (
         return TCG_PP_OPERATION_RESPONSE_BIOS_FAILURE;
       }
 
+      // 

```

Comment:
Did you omit `case TCG2_PHYSICAL_PRESENCE_LOG_ALL_DIGESTS` intentionally?  That operation seems to select all digests and somebody may try using it.